ui/src/createAzureClient.ts

```diff
@@ -5,13 +5,14 @@
 
 import { ServiceClientCredentials } from "ms-rest";
 import { AzureEnvironment, AzureServiceClientOptions } from "ms-rest-azure";
+import * as vscode from "vscode";
 import { IAddUserAgent } from "../index";
 import { addExtensionUserAgent } from "./extensionUserAgent";
 
 export function createAzureClient<T extends IAddUserAgent>(
     clientInfo: { credentials: ServiceClientCredentials; subscriptionId: string; environment: AzureEnvironment; },
     clientType: new (credentials: ServiceClientCredentials, subscriptionId: string, baseUri?: string, options?: AzureServiceClientOptions) => T): T {
-    const client: T = new clientType(clientInfo.credentials, clientInfo.subscriptionId, clientInfo.environment.resourceManagerEndpointUrl);
+    const client: T = new clientType(clientInfo.credentials, clientInfo.subscriptionId, clientInfo.environment.resourceManagerEndpointUrl, { acceptLanguage: vscode.env.language });
     addExtensionUserAgent(client);
     return client;
 }
```

Comment:
Can you add it to `createAzureSubscriptionClient`, too? I assume it works the same way

Reply:
Sure